### DISCUSSION TITLE
Support for default sorting column

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo_admin#280] [Ideas]
Currently, In piccolo admin, every table is sorted according to id , and we can later sort it in the frontend using dropdown.
But, it will be good in normal cases, we have cases in which by default the table has to be updated according to the column  `updated_at`.
So, we have to do the sort, every time we have to watch it. 
It will be easier and also an interesting feature if `TableConfig` can have `sorted_column` as parameter.

**Comment by dantownsend** [comment]
I think it's a good idea - we should probably do this.

**Reply by sumitsharansatsangi** [reply]
Please do it.

**Comment by sinisaos** [comment]
@dantownsend Can I try to do this?

**Reply by dantownsend** [reply]
Yes, of course.

The question is should this be a Piccolo ORM feature, where you define it on the table?

**Reply by sinisaos** [reply]
In the `TableConfig` (similar to the link_column) and then sorting in the UI from the schema. If specified, the rows are sorted by `sort_column`, otherwise the default `primary_key` column is used to sort the rows. I can make a PR for you to see, and if I missed the point, you can close the PR.
